packages/atomic/src/components/search/atomic-sort-dropdown/atomic-sort-dropdown.tsx

```diff
@@ -9,15 +9,15 @@ import {
   SearchStatusState,
 } from '@coveo/headless';
 import {
-  Bindings,
   BindStateToController,
   InitializableComponent,
   InitializeBindings,
 } from '../../../utils/initialization-utils';
```

Comment:
how do you guys feel about having aliases for folders like utils and common? so we can to
`@utils/initialization-utils` instead of `../../../utils/initialization-utils`.
I know Sami removed the aliases in the Qubit project and I don't remember what his reasons were, but I can ask if you guys think it could be an improvement!

Reply:
Worth a try!

Reply:
Yeah good idea !

Reply:
After the internal slack discussion, I would say this this might be worth visiting, but out of scope of this PR.

Reply:
for sure!!